turnpy/turn_integrator.py

```diff
@@ -151,126 +173,139 @@ def send_media_message(msisdn: str, line_name: str, media_type: str, media_id: s
                 {
                     "type": "reply",
-                    "reply": {
-                        "id": button['callback_id'],
-                        "title": button['text']
-                    }
+                    "reply": {"id": button["callback_id"], "title": button["text"]},
                 }
             )
 
-    if interactive_type == 'list':
-        message_data['interactive']['action']['button'] = sections['list_button']
-        message_data['interactive']['action']['sections'] = []
-        message_data['interactive']['action']['sections'].append({})
-        message_data['interactive']['action']['sections'][0]['title'] = sections['list_title']
-        message_data['interactive']['action']['sections'][0]['rows'] = []
-        for list_item in sections['list_items']:
-            message_data['interactive']['action']['sections'][0]['rows'].append(
-                {
-                    "id": list_item['callback_id'],
-                    "title": list_item['text']
-                }
+    if interactive_type == "list":
+        message_data["interactive"]["action"]["button"] = sections["list_button"]
+        message_data["interactive"]["action"]["sections"] = []
+        message_data["interactive"]["action"]["sections"].append({})
+        message_data["interactive"]["action"]["sections"][0]["title"] = sections[
+            "list_title"
+        ]
+        message_data["interactive"]["action"]["sections"][0]["rows"] = []
+        for list_item in sections["list_items"]:
+            message_data["interactive"]["action"]["sections"][0]["rows"].append(
+                {"id": list_item["callback_id"], "title": list_item["text"]}
             )
 
-
-
-    print(message_data)
     response = send_message(line_name, message_data)
-    print(response.text)
+    logging.debug("Sent interactive message response", response.text)
     return response
 
+
 """MEDIA"""
 """
 Save media to Turn for sending.
 
 See the supported file types on the Turn documentation here:
 https://whatsapp.turn.io/docs/api/media#supported-file-types
 """
+
+
 def save_media(line_name: str, type: str, file_binary: str) -> requests.Response:
     auth_headers = {
-        'Authorization': f'Bearer {turn_credentials(line_name)}',
-        'Content-Type': type
+        "Authorization": f"Bearer {turn_credentials(line_name)}",
+        "Content-Type": type,
     }
-    response = requests.post('https://whatsapp.turn.io/v1/media', headers=auth_headers, data=file_binary)
-    print(response.text)
+    response = requests.post(
+        "https://whatsapp.turn.io/v1/media", headers=auth_headers, data=file_binary
+    )
+    logging.debug("Saved media response", response.text)
     return response
 
+
 """CLAIMS"""
 """
 Manage claimed numbers, like determining a claim by a Turn process line a Journey,
 or deleting one.
 
 See: https://whatsapp.turn.io/docs/api/extensions#managing-conversation-claims
 """
+
+
```

Comment:
@thompsgj I am not sure a two line break is needed before and after the comments, lets rather leave it one line above and below.

Reply:
@raouldevil Sure, I've made the adjustments to the spacing around comments.